openedx/features/caliper_tracking/transformers/forum_transformers.py

```diff
@@ -39,3 +39,46 @@ def edx_forum_response_created(current_event, caliper_event):
     })
 
     return caliper_event
+
+
+def edx_forum_comment_created(current_event, caliper_event):
+    """
+    Users create a comment about a response by entering text and then
+    submitting the contributions. When these actions are complete, the
+     server emits an edx.forum.comment.created event.
+
+    :param current_event: default log
+    :param caliper_event: log containing both basic and default attribute
+    :return: final created log
+    """
+
+    current_event_details = current_event['event']
+    current_event_details.pop('url', '')
+
+    caliper_object = {
+        'body': current_event_details.pop('body', ''),
```

Comment:
Why is there a need to specify a default value for body?

Reply:
@asamolion  This case will never run, It is just for precaution. Should i remove it ?

Reply:
Done (Y)